universum/configuration_support.py

```diff
@@ -397,6 +398,9 @@ def __add__(self, other: 'Step') -> 'Step':
         >>> step2 + step1
         {'name': 'barfoo', 'command': ['bar', 'foo'], 'critical': True, 'background': True, 'my_var1': 'barfoo', 'my_var2': 'baz'}
         """
+        if self.is_conditional:
```

Comment:
Cannot change this line to
```
if self.is_conditional or other.is_conditional:
```
because of the following error:
```
2. Processing project configs
 |   ==> Adding file .../pytest-10/test_conditional_true_branch0/artifacts/CONFIGS_DUMP.txt to artifacts...
 |   Error: Exception while processing Universum configuration file:
 |   Traceback (most recent call last):
 |     File ".../Universum/universum/modules/launcher.py", line 385, in process_project_configs
 |       config = self.source_project_configs.filter(check_if_env_set)
 |     File ".../Universum/universum/configuration_support.py", line 721, in filter
 |       item: Step = parent + obj_a
 |     File ".../Universum/universum/configuration_support.py", line 404, in __add__
 |       raise CriticalCiException("Conditional steps addition is not supported yet")
 |   universum.lib.ci_exception.CriticalCiException: Conditional steps addition is not supported yet
 |
 |   Try to execute ``configs.dump()`` to make sure no exceptions occur in that case.
 └ [Failed]
```

Reply:
So do we, in fact, support conditional children?